docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -116,6 +117,17 @@ const copyTextToClipboard = (text: string) => {
   modal.removeChild(textArea);
 };
 
+const parseServerRootPath = (docServiceRoute: string) => {
+  const plainRoute = docServiceRoute.replace('/*', '');
+  const index = window.location.href.indexOf(plainRoute);
+
+  if (index === -1) {
+    return '';
+  }
+  const rootPath = window.location.href.substring(0, index);
+  return rootPath;
+};
```

Comment:
Can we also show an error message if a user bound a `DocService` with other routes than `prefix:`? I think we can throw an exception on the server side, then we could simplify the logic here.

Reply:
Good idea, exact paths don't work (I personally experienced it in the past).

Globs and regex, should we disable them?. I feel like it won't really make sense unless somebody has a particular need (i.e. bound it to a dynamic path).

Reply:
I think so. Let's allow `prefix:` only.